package.json

```diff
@@ -37,39 +36,46 @@
     "@storybook/addon-centered": "^v3.1.2",
     "@storybook/react": "^v3.1.7",
     "babel-core": "^6.4.5",
+    "babel-eslint": "^7.2.3",
     "babel-loader": "^7.1.1",
     "babel-plugin-add-module-exports": "^0.2.1",
-    "babel-plugin-react-remove-prop-types": "^3.0.0",
     "babel-plugin-react-require": "^3.0.0",
     "babel-plugin-react-transform": "^2.0.2",
     "babel-plugin-transform-dead-code-elimination": "^2.0.2",
-    "babel-plugin-transform-inline-environment-variables": "^6.8.0",
+    "babel-plugin-transform-inline-environment-variables": "^0.1.1",
     "babel-plugin-transform-react-constant-elements": "^6.9.1",
     "babel-plugin-transform-react-remove-prop-types": "^0.4.6",
     "babel-preset-es2015-loose": "^8.0.0",
     "babel-preset-react": "^6.5.0",
     "babel-preset-stage-0": "^6.5.0",
     "babel-traverse": "^6.14.0",
     "coveralls": "^2.11.12",
-    "enzyme": "^2.4.1",
+    "enzyme": "^2.9.1",
+    "eslint": "^3.19.0",
+    "eslint-config-airbnb": "^15.0.1",
+    "eslint-config-prettier": "^2.1.1",
+    "eslint-plugin-import": "^2.3.0",
+    "eslint-plugin-jest": "^20.0.3",
+    "eslint-plugin-json": "^1.2.0",
+    "eslint-plugin-jsx-a11y": "^5.0.3",
+    "eslint-plugin-prettier": "^2.1.1",
+    "eslint-plugin-react": "^7.0.1",
     "expect": "^1.20.2",
     "jest-cli": "^20.0.4",
     "jsdom": "^11.1.0",
     "mocha": "^3.0.2",
+    "prettier": "^1.5.2",
     "raw-loader": "^0.5.1",
     "react": "^15.3.1",
-    "react-addons-test-utils": "^15.3.1",
     "react-dom": "^15.3.2",
+    "react-test-renderer": "^15.6.1",
     "react-transform-catch-errors": "^1.0.2",
     "react-transform-hmr": "^1.0.4",
     "redbox-react": "^1.3.0",
-    "source-map-loader": "^0.2.1",
-    "ts-loader": "^0.8.2",
-    "tslint": "^3.15.1",
-    "typescript": "^1.8.10"
+    "source-map-loader": "^0.2.1"
   },
   "dependencies": {
-    "object-assign": "^4.1.0"
+    "babel-cli": "^6.24.1"
```

Comment:
Is `babel-cli` a runtime dependency? If it's only required for build, it should be a devDependency?